bucket/prometheus.json

```diff
@@ -15,9 +16,17 @@
         }
     },
     "bin": [
-        "prometheus.exe",
+        [
+            "prometheus.exe",
+            "prometheus",
+            "--config.file $persist_dir/prometheus.yml"
```

Comment:
You do not create this file. Does it exists inside downloaded archive?

Reply:
Yes, that file is in the archive and contains some sensible default values.